### ISSUE TITLE
Audio sound file naming

### ISSUE THREAD

**Issue by GuillemBelmar** [gtroiani/yucunani_catchgame#19]
This is more of a question, but I've realized that some of the audio files are snd_EN_pu/mx... others are snd_ES_pu/mix... shouldn't that be consistent?

![image](https://user-images.githubusercontent.com/74800434/210398469-09a14d20-401b-4a99-95d3-2a14da6b8162.png)

**Comment by gtroiani** [comment]
yes it should, that was just me creating the files and forgetting how to say the word in one or the other language. the names should all be in English for consistency with the JSON file
